_posts/2021-05-24-ols-3-participant-katharina-kloppenborg-speedblog.md

```diff
@@ -0,0 +1,55 @@
+
+While doing my background research I realized that I struggled to pin down how peer production ideas translate into web design. People collaborate online, there is no one actively assigning tasks and organizing the work centrally, so the online infrastructure is key. But how to design for peer production? What do the platforms of successful projects look like and what features do they use? And is peer production maybe even already practiced in the field of citizen science?
+
+Preparing a research article to outline peer production for web design, with case studies of citizen science online platforms, I wondered about alternative ways to communicate my findings, as well as about a place to share further peer production features I found after my article was finished, and to possibly get input from other researchers, user experience designers or web developers sharing the same interest.
+
+Apart from that, peer production is about openness and collaboration, so why not start with an open science approach right at the start?
+
+![Illustration of what peer-produced research could look like](https://imgur.com/fbzanCT.jpg)
+
+_What peer-produced research could look like (Illustration by the author. You can reuse it under a CC-BY-SA licence)._
+
+# Expectations and how I heard from OLS
+In summer 2020 I had the chance to work for the [Autistica Citizen Science project](https://alan-turing-institute.github.io/AutisticaCitizenScience/) project as a user experience researcher. Georgia, one of the project leaders, participated in the OLS-2 cohort, which got me interested, because I wanted to learn about open science practices at the beginning of my PhD. I joined a cohort call to get a feeling for the program and Georgia and her mentor Anelda kindly invited me to one of their meetings, after which I decided to apply for the next OLS cohort with my own project.
```

Comment:
```suggestion
In summer 2020 I had the chance to work for the [Autistica Citizen Science project](https://alan-turing-institute.github.io/AutisticaCitizenScience/) project as a user experience researcher. [Georgia](community#georgiahca), one of the project leaders, participated in the OLS-2 cohort, which got me interested, because I wanted to learn about open science practices at the beginning of my PhD. I joined a cohort call to get a feeling for the program and Georgia and her mentor [Anelda](/community#anelda) kindly invited me to one of their meetings, after which I decided to apply for the next OLS cohort with my own project.
```